1-index.html

```diff
@@ -44,13 +44,11 @@ <h2>Help needed?</h2>
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Jamulus is Open Source</h2>
-    <p>
-      Ideas? Found a bug? Want to contribute? Since Jamulus is open source you can help us! Find the project on <a href="https://sourceforge.net/projects/llcon/" target="_blank" rel="noreferrer">SourceForge</a> and the <a
-        href="https://github.com/corrados/jamulus" target="_blank" rel="noreferrer">source code on GitHub</a>!
-      Jamulus is licensed under the (<a href="http://www.gnu.org/licenses/gpl-2.0.html" target="_blank" rel="noreferrer">GPL,
-        GNU General Public License</a>).
-    </p>
+      <h2>Contribute</h2>
+      <p>
+        Ideas? Found a bug? Want to contribute? Since Jamulus is <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">free and open source software</a> licensed under the <a href="http://www.gnu.org/licenses/gpl-2.0.en.html" title="GNU General Public License, version 2" target="_blank" rel="noreferrer">GPL</a> you can help us! Find the project on <a href="https://sourceforge.net/projects/llcon/" target="_blank" rel="noreferrer">SourceForge</a> and the <a
```

Comment:
Much better 👍 

I wonder if instead of linking to SF we should expand Contribution.md and link to that so that we can explain the (rather unusual) relationship between SF and GH (that is, SF has the forums for help and initial idea discussion, and GH has code and issue tickets)?

Reply:
Thanks! 👍 

> I wonder if instead of linking to SF we should expand Contribution.md and link to that so that we can explain the (rather unusual) relationship between SF and GH (that is, SF has the forums for help and initial idea discussion, and GH has code and issue tickets)?

I agree.
It would be much better to point to `Contribution.md` and improve and complete this page.
Could be something like:

```html
<p>
Ideas? Found a bug? Want to contribute? Since Jamulus is <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">free and open source software</a> licensed under the <a href="http://www.gnu.org/licenses/gpl-2.0.en.html" title="GNU General Public License, version 2" target="_blank" rel="noreferrer">GPL</a> you can help us!<br/>
Please, take a look at our <a href="wiki/Contribution">Contribution Guidelines</a> to know how to you can help us. You're welcome!
</p>
```

I was just thinking of opening a request for improvement with a few suggestions about `Contribution.md`. ;)

Reply:
Something else, probably off-topic here and to be discussed elsewhere, using the term 'wiki' is no longer appropriate I think.
It was originally a wiki, as I understand it, but now it's a website, with a different procedure for contributing: pages can't be modified directly anymore.

(_sorry again for the off-topic, but my brain works fast and ideas come up as I work, so if I don't talk about it right away I can forget about it and move on to something else_)

Reply:
OK good I'll re-merge that version then.

And yes I've been thinking the same thing about the use of the word "wiki". Was going to ask if we should retire it. I also hope that the current level of change activity will settle down over the next few releases so that we're only making small changes every now and again - but we'll see!